docs/extensions/controllerregistration.md

```diff
@@ -16,10 +16,21 @@ spec:
   resources:
   - kind: OperatingSystemConfig
     type: coreos
+    primary: true
 ```
 
 This information tells Gardener that there is an extension controller that can handle `OperatingSystemConfig` resources of type `coreos`.
-It will now create `ControllerInstallation` resources:
+
+Also, it specifies that this controller is the primary one responsible for the lifecycle of the `OperatingSystemConfig` resource.
+Setting `primary` to `false` would allow to register additional, secondary controllers that may also watch/react on the `OperatingSystemConfig/coreos` resources, however, only the primary controller may change/update the `status` of the extension object.
```

Comment:
> only the primary controller may change/update the `status`

Why is there such a restriction? That is also not possible to enforce?  Secondary controllers must be able to alter the status of the Extension resource (for instance update a condition / add an additional condition) .